### PR TITLE
adds 401s to all: Query 200s

### PR DESCRIPTION
**Pull request by m4sterbunny**

### CODE REVIEW

File: src/openapi.yaml [hunk 1]
```diff
@@ -133,6 +133,12 @@ paths:
                         name:
                           type: string
                           description: User's Name
+          '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:134]: @jehrlich Note, I applied 401s to all endpoints -- I assumed this was valid and matched your comment in Slack. If not, please advise...

File: src/openapi.yaml [hunk 2]
```diff
@@ -158,6 +164,12 @@ paths:
                   updated-records:
                     type: integer
                     description: number of db records altered
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:165]: 401

File: src/openapi.yaml [hunk 3]
```diff
@@ -188,6 +200,12 @@ paths:
                 properties:
                   user:
                     $ref: '#/components/schemas/User'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:201]: 401

File: src/openapi.yaml [hunk 4]
```diff
@@ -217,6 +235,12 @@ paths:
                   deleted:
                     type: integer
                     description: number of Users deleted
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:236]: 401

File: src/openapi.yaml [hunk 5]
```diff
@@ -240,6 +264,12 @@ paths:
                 properties:
                   recorded:
                     type: integer
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:265]: 401

File: src/openapi.yaml [hunk 6]
```diff
@@ -264,6 +294,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/User'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:295]: 401

File: src/openapi.yaml [hunk 7]
```diff
@@ -291,6 +327,12 @@ paths:
                 properties:
                   user:
                     $ref: '#/components/schemas/User'
+        '401':
+          description: password or identifier mismatch
+          content:
+            application/problem+json:
```
Comment (m4sterbunny) [inline src/openapi.yaml:331]: 401

File: src/openapi.yaml [hunk 8]
```diff
@@ -324,6 +366,12 @@ paths:
                   - pxId: 1
                     name: example protocol
                     clonable: false
+          '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:367]: 401

File: src/openapi.yaml [hunk 9]
```diff
@@ -346,6 +394,12 @@ paths:
       responses:
         '204':
           description: no content
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:395]: 401

File: src/openapi.yaml [hunk 10]
```diff
@@ -360,6 +414,12 @@ paths:
...
```
Comment (m4sterbunny) [inline src/openapi.yaml:415]: 401

File: src/openapi.yaml [hunk 11]
```diff
@@ -392,6 +452,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/ProtocolShare'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:453]: 401

File: src/openapi.yaml [hunk 12]
```diff
@@ -416,6 +482,12 @@ paths:
                     type: integer
                     description: number of shares created
                     example: 1
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:483]: 401

File: src/openapi.yaml [hunk 13]
```diff
@@ -446,6 +518,12 @@ paths:
                   un-shared:
                     type: integer
                     description: number of shares deleted
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:519]: 401

File: src/openapi.yaml [hunk 14]
```diff
@@ -477,6 +555,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Protocol'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:555]: 401

File: src/openapi.yaml [hunk 15]
```diff
@@ -499,6 +583,12 @@ paths:
                 properties:
                   protocol:
                     $ref: '#/components/schemas/Protocol'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:583]: 401

File: src/openapi.yaml [hunk 16]
```diff
@@ -520,6 +610,12 @@ paths:
...
```
Comment (m4sterbunny) [inline src/openapi.yaml:610]: 401

File: src/openapi.yaml [hunk 17]
```diff
@@ -540,6 +636,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Eligibility'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:636]: 401

File: src/openapi.yaml [hunk 18]
```diff
@@ -560,6 +662,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Diagnosis'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:662]: 401

File: src/openapi.yaml [hunk 19]
```diff
@@ -580,6 +688,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Protocol'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:688]: 401

File: src/openapi.yaml [hunk 20]
```diff
@@ -603,6 +717,12 @@ paths:
                   _dose:
                     type: string
                     description: string representation of submitted Dosage
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:717]: 401

File: src/openapi.yaml [hunk 21]
```diff
@@ -623,6 +743,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Px'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:743]: 401

File: src/openapi.yaml [hunk 22]
```diff
@@ -651,6 +777,12 @@ paths:
                 properties:
                   protocol:
                     $ref: '#/components/schemas/Protocol'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:777]: 401

File: src/openapi.yaml [hunk 23]
```diff
@@ -675,6 +807,12 @@ paths:
                   deprecated:
                     type: integer
                     description: number of Protocols deprecated
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:807]: 401

File: src/openapi.yaml [hunk 24]
```diff
@@ -695,6 +833,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Prescription'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:833]: 401

File: src/openapi.yaml [hunk 25]
```diff
@@ -710,6 +854,12 @@ paths:
                 properties:
                   prescription:
                     $ref: '#/components/schemas/PrescriptionRequest'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:854]: 401

File: src/openapi.yaml [hunk 26]
```diff
@@ -738,6 +888,12 @@ paths:
                 properties:
                   prescription:
                     $ref: '#/components/schemas/Prescription'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:888]: 401

File: src/openapi.yaml [hunk 27]
```diff
@@ -763,6 +919,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Prescription'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:919]: 401

File: src/openapi.yaml [hunk 28]
```diff
@@ -781,6 +943,12 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/LabelClaim'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:943]: 401

File: src/openapi.yaml [hunk 29]
```diff
@@ -862,6 +1030,12 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/GreenbookHeader'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:1030]: 401

File: src/openapi.yaml [hunk 30]
```diff
@@ -884,6 +1058,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GreenbookHeader'
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:1058]: 401

File: src/openapi.yaml [hunk 31]
```diff
@@ -916,6 +1096,12 @@ paths:
                 description: XML SPL data file served from FDA server
                 type: string
                 format: xml
+        '401':
```
Comment (m4sterbunny) [inline src/openapi.yaml:1096]: 401

**Review by jehrlich — APPROVED** [review]

**Pull request merged**